### DISCUSSION TITLE
You are marked as spam, and therefore cannot authorize a third party application.

### DISCUSSION THREAD

**Discussion by myendmess** [community/community#63563] [New to GitHub]
Good day,
i have this message show up (in object) when i try to submit a project, i don't know what to do it's already 3 days that i'm looking for a solution.
Please it's for school !

**Comment by NuhuWambali** [comment]
same problem i got

**Comment by KantemirMirza** [comment]
when Intelj ide connecting to github , You are marked as spam, and therefore cannot authorize a third party application.

**Comment by zlfteshuai** [comment]
same problem i got

**Comment by krishnenduash** [comment]
same problem i got , how to fix please tell me

**Comment by ghost** [comment]
I experience the same issue. Will there be a fix or any solution?

**Comment by ghostinhershell** [comment]
Hi all 👋🏾 

Responding here so that there is an official answer to this question. This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
